### ISSUE TITLE
"ping" is not installed in container, so startup fails

### ISSUE THREAD

**Issue by Bockiii** [itzg/docker-minecraft-server#891]
Hi,

I found this issue with Enigmatica 2 - Expert Skyblock but maybe this is also an issue for other modpacks.

https://github.com/NillerMedDild/Enigmatica2/blob/1111595b724c757ac695f349551f4531aa3f80cd/ServerStartLinux.sh#L161

The serverstart script has an online check that will ping around to check if there is internet availability. The check fails because ping isn't installed in the container.

There is a workaround (you can set the setting in the config file to disable the internet check) but it definitely requires some research. Adding ping shouldn't cause too much overhead in the container.

**Comment by itzg** [comment]
Yes, that would be quite doable.

As a note for myself:
- Debian : iputils-ping
- Alpine: iputils

**Comment by itzg** [comment]
ping is now included in the latest image, so it's ready to test

**Comment by Bockiii** [comment]
This is fixed btw :) Thanks for adding it.

For some reason, enigmatica 2 expert skyblock has another problem (not sure if its your image or their startscript thats responsible for this): It will not start until you manually download the forge installer.

I think this line isn't doing what it's supposed to: https://github.com/EnigmaticaModpacks/Enigmatica2/blob/master/ServerStartLinux.sh#L113

I used this image: itzg/minecraft-server:java8-openj9

I fixed it by just manually repeating the steps outside of the container. It picked up the installer.jar and everything was fine. But this issue (the ping one) is solved.
